inference-engine/ie_bridges/java/cpp/enum_mapping.hpp

```diff
@@ -0,0 +1,110 @@
+#include <map>
+#include "inference_engine.hpp"
+
+//
+// resize_algorithm
+//
+typedef enum {
```

Comment:
Why should you duplicate c++ enums?

Reply:
We need these mappings to establish an exact match between Java enum and C ++ enum. But after your comment, we made it simpler: now map looks like ```std::map<int, c++_enum_value>```